doc/classes/BaseMaterial3D.xml

```diff
@@ -64,6 +64,7 @@
 		<member name="albedo_texture" type="Texture2D" setter="set_texture" getter="get_texture" keywords="diffuse_texture">
 			Texture to multiply by [member albedo_color]. Used for basic texturing of objects.
 			If the texture appears unexpectedly too dark or too bright, check [member albedo_texture_force_srgb].
+			[b]Note:[/b] When assigned in the editor, other textures' properties will automatically be set based on their texture name. For example, if you assign [code]res://levels/wood_albedo.png[/code] to [member albedo_texture], then [code]res://levels/wood_normal.png[/code] will be assigned to [member normal_texture] and so on. This is based on common texture name conventions, so most texture websites should be handled automatically out of the box. This behavior can be disabled by setting [member EditorSettings.interface/inspector/auto_assign_pbr_material_textures] to [code]false[/code].
```

Comment:
This note is so much. It would be so INSANELY annoying to read and localize for absolutely everything where this applies. I also am not sure if most of this is necessary, given the setting is enabled by default, and the editor tells you about this action happening. I think the information of what naming standards automatically enable this feature need a better place to exist.